db/migrate/20231214142055_create_providers.rb

```diff
@@ -0,0 +1,9 @@
+class CreateProviders < ActiveRecord::Migration[7.1]
+  def change
+    create_table :providers, id: :uuid do |t|
+      t.string :accredited_provider_id, null: false, index: { unique: true }
+
+      t.timestamps
```

Comment:
Based on the erd, I don't think this needs to be scoped to a particular service. Is that right?